dafny-lang/dafny
# Fix for LanguageServerFilesystem

### Changes

1. Correctly handle mixing of on-disc and in-memory files
1. Let verification diagnostics be published in project aware mode

### Tests
1. Add `OnDiskProducerResolutionErrors` test
1. Add `OnDiskProducerVerificationErrors ` test

<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>
